android/app/src/main/java/com/microsoft/reacttestapp/TestApp.kt

```diff
@@ -13,17 +13,19 @@ val Context.testApp: TestApp
     get() = applicationContext as TestApp
 
 class TestApp : Application(), ReactApplication {
+    private lateinit var manifestProviderInternal: ManifestProvider
```

Comment:
Q: what do we need the internal postfix for? :)

Reply:
Ah, I remember now. I think there was a name with variables in ReactApplication.